testing-integration-py/test_s3_it.py

```diff
@@ -0,0 +1,57 @@
+import os
+import unittest
+
+import boto3
+from pulumi.x import automation as auto
+
+from resource_s3 import BUCKET_NAME, OUTPUT_KEY_BUCKET_NAME, OUTPUT_KEY_REGION
+
+
+class TestS3(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls) -> None:
+        cls.STACK_NAME = 'staging'
+        cls.REGION_NAME = 'eu-north-1'
+        cls.WORK_DIR = os.path.join(os.path.dirname(__file__))
+        cls.FILE_NAME = 'bucket.txt'
+
+        cls.stack = auto.create_or_select_stack(stack_name=cls.STACK_NAME, work_dir=cls.WORK_DIR)
+        cls.stack.set_config("aws:region", auto.ConfigValue(value=cls.REGION_NAME))
+        cls.stack.up(on_output=print)
+        cls.outputs = cls.stack.outputs()
+        cls.s3 = boto3.resource('s3')
+
+    @classmethod
+    def tearDownClass(cls) -> None:
+        cls.stack.destroy(on_output=print)
```

Comment:
You might want to remove the stack completely at the end.

```suggestion
        cls.stack.destroy(on_output=print)
        cls.stack.workspace.remove_stack(cls.STACK_NAME)
```